### DISCUSSION TITLE
v3.5.1

### DISCUSSION THREAD

**Discussion by danielroe** [nuxt/nuxt#20983] [Announcements]
> 3.5.1 is the a patch release, with bug fixes and performance improvements.

## 👉 Changelog

[compare changes](https://github.com/nuxt/nuxt/compare/v3.5.0...v3.5.1)

### 🔥 Performance

  - **nuxt:** Use granular watcher to avoid crawling ignored dirs ([#20836](https://github.com/nuxt/nuxt/pull/20836))

### 🩹 Fixes

  - **webpack:** Analyze report path ([#20878](https://github.com/nuxt/nuxt/pull/20878))
  - **nuxt:** Allow island renders without `/` route ([#20894](https://github.com/nuxt/nuxt/pull/20894))
  - **nuxt:** Infer `useFetch` method when generic is passed ([#20797](https://github.com/nuxt/nuxt/pull/20797))
  - **nuxt:** Prioritise vue app context when available ([#20910](https://github.com/nuxt/nuxt/pull/20910))
  - **nuxt:** Don't `refresh` when hydrating when data is present ([#20916](https://github.com/nuxt/nuxt/pull/20916))
  - **nuxt:** Resolve layer assets in relation to layer directory ([#20932](https://github.com/nuxt/nuxt/pull/20932))
  - **nuxt:** Don't match partial component names with prefix ([#20939](https://github.com/nuxt/nuxt/pull/20939))
  - **kit:** Resolve relative module paths when installing ([#20896](https://github.com/nuxt/nuxt/pull/20896))
  - **nuxt:** Exclude plugin declaration from non-existent files ([#20974](https://github.com/nuxt/nuxt/pull/20974))
  - **nuxt:** Use `default` type for initial value for composables ([#20968](https://github.com/nuxt/nuxt/pull/20968))
  - **nuxt:** Skip middleware for islands components ([#20924](https://github.com/nuxt/nuxt/pull/20924))
  - **nuxt:** Use `resolvePath` to handle edge cases for modules ([#20975](https://github.com/nuxt/nuxt/pull/20975))

### 📖 Documentation

  - Fix defaults in custom fetch example ([#20898](https://github.com/nuxt/nuxt/pull/20898))
  - Fix typo ([#20907](https://github.com/nuxt/nuxt/pull/20907))
  - Add `pnpm test` command to run whole test suite ([4907660ff](https://github.com/nuxt/nuxt/commit/4907660ff))
  - Remove warning around `experimental.renderJsonPayloads` ([891ba880e](https://github.com/nuxt/nuxt/commit/891ba880e))
  - Add example of 'alphabetical' plugin numbering ([#20930](https://github.com/nuxt/nuxt/pull/20930))
  - **rendering:** Improve rendering modes section ([244c68108](https://github.com/nuxt/nuxt/commit/244c68108))
  - **rendering:** Lint fix ([ef8b5b593](https://github.com/nuxt/nuxt/commit/ef8b5b593))
  - Sync `useAsyncData` and `useFetch` types ([#20935](https://github.com/nuxt/nuxt/pull/20935))
  - Update static to isr ([#20964](https://github.com/nuxt/nuxt/pull/20964))
  - Add advanced usage example of `useState` ([#20249](https://github.com/nuxt/nuxt/pull/20249))
  - Add link to layers from `pages/` docs ([#20976](https://github.com/nuxt/nuxt/pull/20976))

### 🏡 Chore

  - Rename script test for fixtures:payload ([#20908](https://github.com/nuxt/nuxt/pull/20908))
  - Pass ref to checkout ([58ed8ec34](https://github.com/nuxt/nuxt/commit/58ed8ec34))
  - Remove outdated todo comment ([b2a769a86](https://github.com/nuxt/nuxt/commit/b2a769a86))

### 🤖 CI

  - Try different ref format ([2b10dd7b7](https://github.com/nuxt/nuxt/commit/2b10dd7b7))

### ❤️  Contributors

- Paul Grau (@graup)
- Luciano Tonet (@lucianotonet)
- Daniel Roe (@danielroe)
- Julien Huang (@huang-julien)
- Haruaki OTAKE (@aaharu)
- Nate Emerson (@nateemerson)
- Jongmin Yoon (@jongmin4943)
- Sébastien Chopin (@Atinux)
- Aziz Ramdan Kurniawan (@azizramdan)
- @Gypsophyllite 
- Wilson Pinto (@wilsonpinto)
- Lemon (@ww-lemon)
- Vasily Kuzin (@ExEr7um)
- Anthony Fu (@antfu)

<hr /><em>This discussion was created from the release <a href='https://github.com/nuxt/nuxt/releases/tag/v3.5.1'>v3.5.1</a>.</em>

**Comment by FayKn** [comment]
Anyone else also getting this error, if so how do I fix it?


![image](https://github.com/nuxt/nuxt/assets/61050421/3273dcea-10d2-47cf-8f62-050400434b13)



this is my package.json

```
{
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev --host 0.0.0.0",
    "generate": "nuxt generate",
    "preview": "nuxt preview"
  },
  "devDependencies": {
    "@nuxt/devtools": "^0.3.1",
    "@nuxt/image-edge": "^1.0.0-27840416.dc1ed65",
    "@nuxtjs/color-mode": "^3.1.7",
    "@nuxtjs/tailwindcss": "^6.6.6",
    "@tailwindcss/aspect-ratio": "^0.4.0",
    "@tailwindcss/forms": "^0.5.1",
    "@tailwindcss/typography": "^0.5.2",
    "autoprefixer": "^10.4.7",
    "nuxt": "^3.5.1",
    "postcss": "^8.4.14"
  },
  "dependencies": {
    "h3": "^0.8.5",
    "nuxt-lazy-load": "^1.2.9",
    "ohmyfetch": "^0.4.18",
    "untyped": "^0.5.0"
  }
}
```

**Reply by jongmin4943** [reply]
It sometimes happened to me when I run `nuxi upgrade --force` on `windows os`. The workaround that works for me is that delete `.nuxt`, `pnpm-lock.yaml` file and `node_modules` first, and then, run `pnpm install`.
